src/SWP/Bundle/ElasticSearchBundle/Loader/SearchResultLoader.php

```diff
@@ -110,6 +110,8 @@ public function load($metaType, $withParameters = [], $withoutParameters = [], $
             }
         }
 
+        $metaCollection->suggestedTerm = $repository->getSuggestedTerm($criteria->getTerm());
```

Comment:
we need some behat tests covering this part (in behat you have elastic running, and twig template can be tested)